include/node.tcc

```diff
@@ -653,6 +653,7 @@ void mpm::Node<Tdim, Tdof,
   node_mutex_.unlock();
 }
 
+//! TwoPhase functions--------------------------------------------------------
```

Comment:
Could we move these to a separate file, so it's easier?

Reply:
Just separate `node.tcc` or both `node.tcc` and `node.h`?